ExotischNederland/Menus/GameMenu.cs

```diff
@@ -0,0 +1,211 @@
+                    CreateGame();
+                }
+                else if (selected == "editGame")
+                {
+                    EditGame();
+                }
+                else if (selected == "deleteGame")
+                {
+                    DeleteGame();
+                }
+                else if (selected == "playGame")
+                {
+                    PlayGame();
+                }
+                else if (selected == "back")
+                {
+                    break;
+                }
+            }
+        }
+
+        private void CreateGame()
+        {
+            Console.Clear();
+            Console.WriteLine("Create a New Game");
+            Console.Write("Enter the title of the game: ");
+            string title = Console.ReadLine();
+            Console.Write("Enter the description of the game: ");
+            string description = Console.ReadLine();
+
+            Console.Write("Enter route ID for the game: ");
+            int routeId;
+            if (int.TryParse(Console.ReadLine(), out routeId))
+            {
+                Route route = Route.Find(routeId);
+                if (route != null)
+                {
+                    Game game = Game.Create(route, title, description);
+                    Console.WriteLine($"Game '{title}' created successfully.");
+                }
+                else
+                {
+                    Console.WriteLine("Invalid route ID. Game creation failed.");
+                }
+            }
+            else
+            {
+                Console.WriteLine("Invalid route ID input.");
+            }
+
+            Console.WriteLine("Press any key to return to the menu.");
+            Console.ReadKey();
+        }
+
+        private void EditGame()
+        {
+            Console.Clear();
+            Console.WriteLine("Edit a Game");
+            Console.Write("Enter the ID of the game to edit: ");
+            int gameId;
+            if (int.TryParse(Console.ReadLine(), out gameId))
+            {
+                Game game = Game.Find(gameId);
+
+                if (game != null)
+                {
+                    Console.Write("New title (leave empty to keep current): ");
+                    string newTitle = Console.ReadLine();
+                    Console.Write("New description (leave empty to keep current): ");
+                    string newDescription = Console.ReadLine();
+
+                    game.Update(newTitle, newDescription);
+                    Console.WriteLine("Game updated successfully.");
+                }
+                else
+                {
+                    Console.WriteLine("Game not found.");
+                }
+            }
+            else
+            {
+                Console.WriteLine("Invalid game ID input.");
+            }
+
+            Console.WriteLine("Press any key to return to the menu.");
+            Console.ReadKey();
+        }
+
+        private void DeleteGame()
```

Comment:
You would need to know the ID of the game to delete, with the risk of entering the wrong one. It would be safer to navigate to the game and delete it from the details page. See the ObservationMenu for an example of that